django/db/models/sql/compiler.py

```diff
@@ -957,7 +1007,7 @@ def execute_sql(self, result_type):
             if is_empty:
                 rows = aux_rows
                 is_empty = False
-        return rows
+        return (rows, returning_values)
```

Comment:
Frankly, I don't remember the details here, and I only have time to glimpse the PR now, but it seems that there may be two cases where multiple rows of returned values may be expected: `bulk_create()` on one hand, and an `UPDATE` statement on the other (you wouldn't need to returned values in an ORM `update()` call, but you would need them in the lower-level API).